public-api/src/test/java/org/pmiops/workbench/cdr/dao/SurveyModuleDaoTest.java

```diff
@@ -0,0 +1,73 @@
+package org.pmiops.workbench.cdr.dao;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.Assert;
+import java.util.List;
+import org.junit.runner.RunWith;
+import org.pmiops.workbench.cdr.model.DbSurveyModule;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.liquibase.LiquibaseAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+@Import(LiquibaseAutoConfiguration.class)
+@AutoConfigureTestDatabase(replace= AutoConfigureTestDatabase.Replace.NONE)
```

Comment:
If you add a spring property `liquibase.enabled=false` in your src/test/resources/application.properties file you can remove these annotations. Example in workbench: https://github.com/all-of-us/workbench/blob/3e30c47b8a91539e1dfbbe8854e09b5ed7d5bd3d/api/src/test/resources/application.properties#L7

Reply:
Thanks. Removed them.